File: Pb-2/pb-2.go
package main
import "fmt"

func fiboEvenSum(num int) int{
  a, b, sum := 0, 2, 0
  for b < num{
    sum += b
    a, b = b, a + 4 * b
  }
  return sum
}

func main(){
  fmt.Println(fiboEvenSum(43))
}

File: pb-2.py
def fiboEvenSum(n):
  n1=0
  n2=1
  count=0
  sum=0
  while count<n:
      count=n1+n2
      n1=n2
      n2=count
      if count%2==0:
        sum+=count
  return sum

if __name__== "__main__":
  print(fiboEvenSum(10))
